prov/gni/src/gnix_vc.c

```diff
@@ -1235,6 +1306,7 @@ int _gnix_vc_connect(struct gnix_vc *vc)
 	 */
 
 	fastlock_acquire(&cm_nic->wq_lock);
+	vc->modes |= GNIX_VC_MODE_IN_CM_WQ;
```

Comment:
It turns out I don't think we really need to set a bit to indicate the vc is in the CM_WQ.  I'll remove this bit setting.